test/ihe/pixm.js

```diff
@@ -309,10 +281,45 @@ const requestAndAssertOperationOutcome = (tp, t, done) => {
     t.equal(body.issue[0].severity, tp.severity)
     t.equal(body.issue[0].code, tp.code)
     t.equal(body.issue[0].details.text, tp.details)
+    t.equal(body.issue[0].diagnostics, tp.diagnostics)
     done()
   })
 }
 
+tap.test('PIXm Query, should return 404 when recognises the domain but can\'t find the id', (t) => {
+  // Given
+  basicPIXmTest(t, (db, done) => {
+    const testParams = {
+      statusCode: 404,
+      sourceIdentifier: 'sourceIdentifier=pshr:sanid|non-existent-id',
+      targetSystem: '',
+      severity: 'error',
+      code: 'not-found',
+      details: 'sourceIdentifier Patient Identifier not found',
+      diagnostics: 'sourceIdentifier Patient Identifier not found'
+    }
+
+    requestAndAssertOperationOutcome(testParams, t, done)
+  })
+})
+
+tap.test('PIXm Query, should return 404 when recognises the domain but can\'t find the id', (t) => {
```

Comment:
This test seems to be a duplicate of the one above?

Reply:
It's using a slightly different source identifier, I think it's testing full URL assigning authorities. It was already part of the tests so I figured I'd leave it around.

Reply:
ok cool lets keep it in then to illustrate that we cater for full URLs as identifiers